src/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 2.6)
 
 option(EGL "Set to ON if targeting an EGL device" ${EGL})
 option(PANDORA "Set to ON if targeting an OpenPandora" ${PANDORA})
-option(MUPENPLUSAPI "Set to ON for Mupen64Plus plugin" ${MUPENPLUSAPI})
+option(MUPENPLUSAPI "Set to ON for Mupen64Plus plugin" ${UNIX})
```

Comment:
Huh, GitHub doesn't seem to have updated what's it's showing here, even though the branch was updated. Can you re-open the pull request, or should I make another one?